src/routes/api/data-cleanup/cleanup-reposyncstates.test.ts

```diff
@@ -21,69 +21,16 @@ describe("Cleanup RepoSyncState", () => {
 			installationId: 123
 		});
 		await createdRepoSyncState("first", sub.id);
-		await createdRepoSyncState("second", 9999);
-		expect([...await RepoSyncState.findAll()].length).toBe(2);
+		expect([...await RepoSyncState.findAll()].length).toBe(1);
 
 		//call clean up api
 		await supertest(app).delete(`/api/data-cleanup/repo-sync-states?repoSyncStateId=9999`).set("X-Slauth-Mechanism", "test").expect(200);
 		await supertest(app).delete(`/api/data-cleanup/repo-sync-states?repoSyncStateId=9999&commitToDB=`).set("X-Slauth-Mechanism", "test").expect(200);
 		await supertest(app).delete(`/api/data-cleanup/repo-sync-states?repoSyncStateId=9999&commitToDB=false`).set("X-Slauth-Mechanism", "test").expect(200);
 		await supertest(app).delete(`/api/data-cleanup/repo-sync-states?repoSyncStateId=9999&commitToDB=whatever`).set("X-Slauth-Mechanism", "test").expect(200);
 
-		//check result is correct
-		expect([...await RepoSyncState.findAll()].length).toBe(2);
-	});
-
-	it("should delete orphan RepoSyncState, while leave rest untouch", async () => {
```

Comment:
These tests can't be run due to the FK. But we still need the api to do a final cleanup right before we run the db migration on prod.